app/modules.go

```diff
@@ -284,6 +284,14 @@ func orderInitBlockers() []string {
 		paramstypes.ModuleName,
 		upgradetypes.ModuleName,
 		vestingtypes.ModuleName,
+		// The globalfee module should ideally be initialized before the genutil module in theory:
+		// The globalfee antehandler performs checks in DeliverTx, which is called by gentx.
+		// When the global fee > 0, gentx needs to pay the fee. However, this is not expected,
+		// (in our case, the global fee is initialized with an empty value, which might not be a problem
```

Comment:
I think this detail can be removed since more context can be found in the #2489 issue.